### PR TITLE
get all users

### PR DESCRIPTION
**Pull request by RansfordGenesis**

### CODE REVIEW

**Review by JerryAgbesi — APPROVED** [review]

**Review by blitzblade — CHANGES_REQUESTED** [review]
Add tests for this

File: test/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+def test_admin_pagination(client):
+    res=client.post(
```
Comment (blitzblade) [inline test/test_admin.py:3]: Why is the request a post request when the endpoint is a get endpoint?

File: test/test_admin.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+      "/api/v1/admin/users/"
+    )
+    assert res.json().status_code == 202
+    assert res.json()
```
Comment (blitzblade) [inline test/test_admin.py:7]: Your test should check for an expected record you know is there.
You can create a user for instance, and then try fetching and asserting that the name of the first user item matches the one you created.

File: test/test_users.py [hunk 3]
```diff
@@ -37,4 +38,32 @@ def test_log_in(client,test_user):
...
+          "password": "food", 
+          "password_confirmation": "food"
+        }
+    )
+    new_user = user.UserResponse(**post_request.json())
+    
+    get_request = client.get("/api/v1/users")
+    
+    # topology of the final response unknown yet.
+    # uncertain if status_code will be included with the list of users
```
Comment (blitzblade) [inline test/test_users.py:61]: Did you run this test locally to see if there are any errors?

Looks like you're trying to fetch status code from a dictionary

**Review by blitzblade — APPROVED** [review]

**Pull request closed without merging**